packages/near-lake-primitives/src/types/block.ts

```diff
@@ -64,6 +108,9 @@ export class Block {
         return logs
```

Comment:
This method is not annotated. 

It returns raw logs regardles of the fact that they  are standard events nor not

![image](https://github.com/near/near-lake-framework-js/assets/25015977/ec8993ac-0b91-4d6f-b62e-a90f0cc06386)

Reply:
Added it 👍